src/main/java/leafout/backend/persistence/ActivityRepository.java

```diff
@@ -0,0 +1,62 @@
+package leafout.backend.persistence;
+
+import leafout.backend.model.Activity;
+import org.springframework.data.mongodb.repository.MongoRepository;
+import org.springframework.stereotype.Repository;
+
+import java.util.List;
+import java.util.Optional;
+import java.util.UUID;
+
+/**
+ * This class represent the repository of a Activity
+ *
+ * @author Juan Pablo Ospina Henao
+ * @since 0.0.1
+ */
+@Repository
+public interface ActivityRepository extends MongoRepository<Activity, String> {
+
+    /**
+     * find all activities created
+     * @return list<Activity>
+     */
+    List<Activity> findAll();
```

Comment:
You don't need to define this method here because `MongoRepository` extends from `CrudRepository`